src/main/webapp/WEB-INF/tv_show_dedicated_page.jsp

```diff
@@ -0,0 +1,122 @@
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                </div>
+                <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                    <ul class="nav navbar-nav">
+                        <li>
+                            <a href="index.html" class="nowPlaying">Home</a>
+                        </li>
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                Genres<span class="caret"></span>
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a href="#" class="action" id="action">Action</a></li>
+                                <li><a href="#" class="animation" id="animation">Animation</a></li>
+                                <li><a href="#" class="comedy" id="comedy">Comedy</a></li>
+                                <li><a href="#" class="drama" id="drama">Drama</a></li>
+                                <li><a href="#" class="family" id="family">Family</a></li>
+                                <li><a href="#" class="fantasy" id="fantasy">Fantasy</a></li>
+                                <li><a href="#" class="horror" id="horror">Horror</a></li>
+                                <li><a href="#" class="music" id="music">Music</a></li>
+                                <li><a href="#" class="romance" id="romance">Romance</a></li>
+                                <li><a href="#" class="scifi" id="scifi">Science Fiction</a></li>
+                                <li><a href="#" class="thriller" id="thriller">Thriller</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                    <ul class="nav navbar-nav navbar-right">
+                        <form class="navbar-form navbar-right searchForm">
+                            <div class="form-group">
+                                <input id="searchMovieNeWindowxfxInput" type="text" class="form-control" placeholder="Search movies">
+                            </div>
+                            <button type="submit" class="btn btn-default">
+                                <i class="glyphicon glyphicon-search"></i>
+                            </button>
+                        </form>
+                        <li class="icon-bar">
+                            <div class="g-signin2" data-onsuccess="onSignIn" data-width="120" data-height="45"></div>
+                        </li>
+                        <li class="icon-bar">
+                            <div class="sign-out">
+                                <button id="signout-button" onclick="signOut()">Sign Out</button>
+                            </div>
+                        </li>
```

Comment:
If you are using the complete nav bar, add the latest change for user details as well.